service/directors.go

```diff
@@ -31,6 +31,13 @@ func recipeAPIDirector(apiRouterVersion string) func(req *http.Request) {
 	}
 }
 
+func uploadServiceAPIDirector(apiRouterVersion string) func(req *http.Request) {
+	return func(req *http.Request) {
+		director(req)
+		req.URL.Path = fmt.Sprintf("/%s%s", apiRouterVersion, req.URL.Path)
```

Comment:
the apiRouterVersion is required only if we proxy calls to the API router. 
If we don't do it, then you can ignore the apiRouterVersion, and just use the base director (without modifying the URL path)